tests/testthat/test-transform_quantile_model_out.R

```diff
@@ -39,59 +40,74 @@ test_that("model_out_tbl_1 has any rows", {
   model_out_tbl_1 <- utils::read.csv(
     test_path("testdata/model_out_tbl_quantile_1.csv")
   )
-  target_observations_1 <- utils::read.csv(
+  oracle_output_1 <- utils::read.csv(
     test_path("testdata/target_data_1.csv")
-  )
+  ) |>
+    dplyr::rename(oracle_value = observation)
   expect_error(
     suppressWarnings(transform_quantile_model_out(
       model_out_tbl = model_out_tbl_1[0, ],
-      target_observations = target_observations_1
+      oracle_output = oracle_output_1
     )),
     regexp = "Must have at least 1 rows, but has 0 rows."
   )
 })
 
 
-test_that("many-to-one relationship exists between model_out_tbl and target_observations", {
+test_that("many-to-one relationship exists between model_out_tbl and oracle_output", {
   model_out_tbl_1 <- utils::read.csv(
     test_path("testdata/model_out_tbl_quantile_1.csv")
   )
-  target_observations_2 <- utils::read.csv(
+  oracle_output_2 <- utils::read.csv(
     test_path("testdata/target_data_2.csv")
-  )
+  ) |>
+    dplyr::rename(oracle_value = observation)
 
   expect_error(
     suppressMessages(transform_quantile_model_out(
       model_out_tbl = model_out_tbl_1,
-      target_observations = target_observations_2
+      oracle_output = oracle_output_2
     )),
     regexp = "Each row in `x` must match at most 1 row in `y`."
   )
 })
 
 test_that("hubExamples data set is transformed correctly", {
-  # forecast_outputs.rda & forecast_target_observations.rda are stored in hubExamples:
+  # forecast_outputs.rda & forecast_oracle_output.rda are stored in hubExamples:
   # https://github.com/hubverse-org/hubExamples/tree/main
-  load(test_path("testdata/forecast_outputs.rda")) # sets forecast_outputs
-  load(test_path("testdata/forecast_target_observations.rda")) # sets forecast_target_observations
-  model_out_tbl <- forecast_outputs
-  target_observations <- forecast_target_observations
+  model_out_tbl <- hubex_forecast_outputs()
+  oracle_output <- hubex_forecast_oracle_output()
   act_forecast <- transform_quantile_model_out(
     model_out_tbl = model_out_tbl,
-    target_observations = target_observations
+    oracle_output = oracle_output
   )
 
-  exp_forecast <- utils::read.csv(
-    test_path("testdata/exp_forecast_hubExamples_2.csv")
```

Comment:
Should this file (and `exp_forecast_hubEsxamples_1.csv`) be removed? It's not being used anywhere anymore.

Reply:
done!